src/Shouldly.Tests/ConventionTests/ApprovePublicApi.ShouldlyApi.approved.cs

```diff
@@ -583,12 +561,10 @@ public static void ShouldBeGreaterThanOrEqualTo<T>([System.Diagnostics.CodeAnaly
             "stomMessage.", true)]
         public static void ShouldBeGreaterThanOrEqualTo<T>(this T actual, T expected, System.Collections.Generic.IComparer<T> comparer, System.Func<string?>? customMessage) { }
         public static void ShouldBeGreaterThanOrEqualTo<T>([System.Diagnostics.CodeAnalysis.AllowNull] this T actual, [System.Diagnostics.CodeAnalysis.AllowNull] T expected, System.Collections.Generic.IComparer<T> comparer, string? customMessage = null) { }
-        public static void ShouldBeInRange<T>(this T actual, T from, T to)
-            where T : System.IComparable<T> { }
         [System.Obsolete("Func based customMessage overloads have been removed. Pass in a string for the cu" +
             "stomMessage.", true)]
         public static void ShouldBeInRange<T>(this T actual, T from, T to, System.Func<string?>? customMessage) { }
-        public static void ShouldBeInRange<T>([System.Diagnostics.CodeAnalysis.DisallowNull] this T actual, [System.Diagnostics.CodeAnalysis.AllowNull] T from, [System.Diagnostics.CodeAnalysis.AllowNull] T to, string? customMessage)
+        public static void ShouldBeInRange<T>([System.Diagnostics.CodeAnalysis.DisallowNull] this T actual, [System.Diagnostics.CodeAnalysis.AllowNull] T from, [System.Diagnostics.CodeAnalysis.AllowNull] T to, string? customMessage = null)
```

Comment:
I wonder why were NRT attributes on one overload and not the other.

Reply:
Followed this to https://github.com/shouldly/shouldly/pull/687/files#r506340416.